sections/02_intro.md

```diff
@@ -25,164 +25,46 @@ could learn to identify cats simply by watching online videos
-the data. Moreover, deep networks are more robust and trainable when employing
-several architectural innovations: weight replication, better behaved
-non-linearities like rectified-linear units, residual networks, and better
-weight initialization, and persistent memory. `TODO: excessive jargon` Likewise
-the central role of dimensionality reduction as a strength of neural networks
-was elucidated, and this has motivated designs built to capitalize on this
-strength `TODO: cite autoencoders and word2vec`.
-
-Third, several advances in training algorithms have enabled applications of
-neural networks in ways not obviously possible. The number of training
-strategies for deep learning is growing rapidly and a complete review is beyond
-our scope. But these algorithms can train networks in domains where earlier
-algorithms struggled.  For example, newer optimizers can very efficiently learn
-using batched training, where only a portion of the data needs to be processed
-at a time. These optimizers more effectively optimize very large weight vectors
-where many weights are only rarely updated. `TODO: awkward phrasing` Noise
-contrastive error has proven particularly useful in modeling language.
-Reinforcement learning has enabled neural networks to learn how to play games
-like chess, GO, and poker. Curriculum learning enables networks to gradually
-build up expertise to solve particularly challenging algorithmic problems.
-Dropout nodes and layers make networks much more robust, even when the number of
-weights are dramatically increased. `TODO: jargon`
-
-Fourth, the convergence of these factors currently makes deep learning extremely
-adaptable, and capable  of addressing the nuanced differences of each domain to
+possible. Easy-to-use software packages including Theano [@arxiv:1605.02688],
+TensorFlow [@arxiv:1605.08695], and Keras
+[@url:https://github.com/fchollet/keras/], have brought the techniques of the
+field out of the specialist's toolkit to a broad community of computational
+scientists. With the growing attention focused on these methods, new techniques
+for fast training have enabled their application to larger datasets
+[@arxiv:1106.5730]. The growing popularity of the field has also led to a
+proliferation of new approaches, many of which are well suited for addressing
+distinct challenges. Finally, the larger datasets now available are also well
+suited to fitting the many parameters that exist for deep neural networks.
+
+The convergence of these factors currently makes deep learning extremely
+adaptable, and capable of addressing the nuanced differences of each domain to
 which it is applied. Many of the advances in deep learning were first developed
-for image analysis and text analysis, but the lessons and techniques learnt
-there enable the construction of very powerful models specifically suited to the
-challenges presented by  each unique domain.
-
-### Deep learning in scientific inquiry
```

Comment:
I am supportive of this. About to push another commit that may address the rest of your points.